### DISCUSSION TITLE
Manual limit on gpu

### DISCUSSION THREAD

**Discussion by notChiche** [Lolliedieb/lolMiner-releases#549] [Q&A]
Hi there! Is there any option or setting to limit the usage on my gpu from de miner itself? I want to be able to work while mining (perhaps limiting its power of mine at 90% or so)

I know I can limit my gpu power form Afterburner, but I dont think that would be a good option for my purpose....

Any help would be considered! Newbie over here.

Best regards

PS: I have an AMD RX 580 8gb

**Comment by qt7621** [comment]
wow, i have the same question, but i see, nobody did answer your question, this is so sad...
